src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -97,6 +97,8 @@ def _ensure_monitor_with_config(
         )
         if created:
             signal_monitor_created(project, None, True)
+        if monitor.project_id != project.id:
```

Comment:
```suggestion

        # TODO(rjo100): Temporarlily logging this since we failed to correctly scope 
        # the Monitor lookups to the DSN's project_id. Meaning you can actually use 
        # any DSN to check-in for any monitor within your organiztion.
        #
        # For now let's see how many of these we get before removing this and 
        # correctly lmiting things
        if monitor.project_id != project.id:
```